src/main/js/api/sse.js

```diff
@@ -52,22 +50,22 @@ exports.connect = function(browser, done) {
  */
 exports.disconnect = function(onDisconnected) {
     function clientDisconnect() {
-        sseClient.disconnect();
+        if (sseConnection) {
+            sseConnection.disconnect();
```

Comment:
Maybe it would be better that  ```sseConnection.disconnect();``` would accept a ```callback``` argument and we would change it to ``` sseConnection.disconnect(onDisconnected);```